syncer/checkpoint.go

```diff
@@ -553,6 +557,43 @@ func (cp *RemoteCheckPoint) FlushPointsExcept(tctx *tcontext.Context, exceptTabl
 	return nil
 }
 
+// FlushPointWithTableInfo implements CheckPoint.FlushPointWithTableInfo
+func (cp *RemoteCheckPoint) FlushPointWithTableInfo(tctx *tcontext.Context, schema string, table string, ti *model.TableInfo) error {
+	cp.Lock()
+	defer cp.Unlock()
+
+	sqls := make([]string, 0, 10)
```

Comment:
Why cap is 10? It sames that we only need 1.
```suggestion
	sqls := make([]string, 0, 1)
```